src/purchases.ts

```diff
@@ -0,0 +1,716 @@
+import { NativeEventEmitter, NativeModules } from "react-native";
+import { PurchasesErrorCode, PurchasesErrorHelper } from "./errors";
+import { PurchaserInfo } from "./purchaserInfo";
+import {
+    PRORATION_MODE,
+    PACKAGE_TYPE,
+    INTRO_ELIGIBILITY_STATUS,
+    PurchasesOfferings,
+    PurchasesProduct,
+    UpgradeInfo,
+    PurchasesPaymentDiscount,
+    PurchasesPackage,
+    IntroEligibility,
+    PurchasesDiscount
```

Comment:
could also have replaced this with an import *, but this is a bit cleaner IMO since there could be stuff in offerings that we don't strictly need to import here